rustworkx/visualization/graphviz.pyi

```diff
@@ -10,17 +10,68 @@ import typing
 from rustworkx.rustworkx import PyGraph, PyDiGraph
 
 if typing.TYPE_CHECKING:
-    from PIL.Image import Image  # type: ignore
+    from typing_extensions import TypeAlias
+    from PIL.Image import Image
+
+    _Method: TypeAlias = typing.Literal["twopi", "neato", "circo", "fdp", "sfdp", "dot"]
```

Comment:
`typing.Literal` can conflict with some users that have `str` from a dictionary or some kind of function. Also, we'd need to stay up to date with graphviz to know what arguments are valid. I'd rather not care.

Reply:
We can additionally allow image_type and method to support str